src/corehost/corehost.cpp

```diff
@@ -102,7 +94,7 @@ int exe_start(const int argc, const pal::char_t* argv[])
 	
 #if FEATURE_APPHOST
     pal::string_t embedded_app_name;
-    if (!is_exe_enabled_for_execution_hack(&embedded_app_name))
+    if (!is_exe_enabled_for_execution(&embedded_app_name))
```

Comment:
So the apphost will look just like it does without bundle (including the special stamped path to the app.dll to run), right?

Reply:
Yes, no change to this mechanism.